proteomic_aggregation/python/proteomic_aggregation.ipynb

```diff
@@ -0,0 +1,3591 @@
+{
```

Comment:
<p>This is a really clear and concise explanation, nice job.</p><p> </p><p>In the first sentence, do you mean to reference the <code>iterate_file_extract</code> function?  I don't see a <code>tsx_extract</code> function being called here.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>